examples/workflow/pygrb/ER8/er8_main_offline.ini

```diff
@@ -113,25 +111,30 @@ num-auto-chisq-points = 40
 auto-veto-time-step = 0.001
 num-chi-square-bins = 16
 chi-square-threshold = 6
-approximant = SpinTaylorFrameless
+approximant = SpinTaylorT4
 order = threePointFivePN
 sngl-snr-threshold = 4
 psd-segment-duration = 256
 do-clustering =
 cluster-window = 0.1
 inverse-spec-length = 32
-face-on-analysis =
-face-away-analysis =
 
-[inspiral-no_injections]
-;do-short-slides =
-;short-slide-offset = 6
+[inpiral-sngl_no_injections&inspiral-sngl_injections]
+do-sngl-chi-tests =
+
+[inspiral-coherent_no_injections]
+do-short-slides =
+short-slide-offset = 6
```

Comment:
Ah balls, this option is needed only for the coherent *and* no injections case. Okay, maybe the way you do it is better for now until we can find a nice way of specifying that an option should only be specified if *two* tags are present.